scormxblock/static/html/scormxblock.html

```diff
@@ -1,31 +1,31 @@
 {% load i18n %}
-<div class="themeable-xblock scormxblock_block">
-    <div class="block-header-wrapper scorm-header-wrapper">
-        <h3 class="scorm-header">{{ display_name }}</h3>
-        {% if has_score_value %}
-        <p class="scorm_process">
-            <span class="fal fa-bullseye-pointer"></span>
-            <span class="lesson_score">{{ scorm_score_value }}</span>
-            <span class="static_score">/{% blocktrans count weight_value as num_points %}{{ num_points }} point possible{% plural %}{{num_points }} points possible{% endblocktrans %} ({% trans graded_status %})</span>
-            <span class="success_status">{{ scorm_status_value }}</span>
-        </p>
-        {% endif %}
+
+<%!
+import cgi
+%>
+<%  
+	player_config_attrs = ''
+%>	
+
+<% 
+for item in list({player_config}.items()):
+    player_config_attrs += 'data-{{}}="{{}}" '.format(item[0], cgi.escape(item[1], quote=True))
+%>
+
+<div class="scorm_xblock">
+    <h3 class="scorm_name">{self.display_name} (External Resource) <span class="scorm_weight">{self.weight} points possible</span></h3>
+    <div class="scorm_description">{self.description}
+        <div class="scorm_launch"><button id="scorm-launch-{self.url_name}">Launch module</button></div>
```

Comment:
- I think that the vars of `self.url_name` or `self.description` are referenced from Method [self.get_student_data()](https://github.com/Learningtribes/edx_xblock_scorm/blob/25ad53e0a912922ce6fdf5c624f34364e1aa4e45/scormxblock/scromxblockng.py#L436)

Reply:
https://github.com/Learningtribes/edx_xblock_scorm/pull/29/commits/45da7854755971b86a11e82c186289635c5d8e55